code/API_definitions/qod-api.yaml

```diff
@@ -481,14 +469,26 @@ components:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
           example:
+            status: "404"
             code: NOT_FOUND
             message: "Session Id does not exist"
+    Generic500:
+      description: Server error
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ErrorInfo"
+          example:
+            status: "500"
+            code: INTERNAL
+            message: "Session could not be deleted"
```

Comment:
This message is not generic to all the procedures.

Reply:
You are right. In the generic model, example should show a more neutral message, I will correct it. Implementations may adapt the text to the scenario, as with other errors.

Reply:
This one is done ... will resolve the conversation.